specification/servicefabric/data-plane/Microsoft.ServiceFabric/6.0/servicefabric.json

```diff
@@ -0,0 +1,11892 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Service Fabric Client APIs",
+    "description": "Service Fabric REST Client APIs allows management of Service Fabric clusters, applications and services.",
+    "version": "6.0.*"
+  },
+  "host": "localhost:19080",

```

Comment:
hot is hardcoded to `localhost:19080`, does this mean the generated client can be used only from within the cluster?

Reply:
No I thought that's only when testing the Swagger spec. The generated client accepts any host.